### DISCUSSION TITLE
[Request] FlareSolverr

### DISCUSSION THREAD

**Discussion by opariffazman** [tteck/Proxmox#2495] [Ideas]
I believe [FlareSolverr](https://github.com/FlareSolverr/FlareSolverr) would be a very nice addition to complement the *arr suites. :)

**Comment by navels** [comment]
FYI I just added FlareSolverr to my Jackett container.  I created a fresh Jackett container with 8GB disk and 1GB RAM and did this:
```
# install chrome
apt install apt-transport-https curl gpg xvfb -y
curl -fSsL https://dl.google.com/linux/linux_signing_key.pub | gpg --dearmor | tee /usr/share/keyrings/google-chrome.gpg >> /dev/null
echo deb [arch=amd64 signed-by=/usr/share/keyrings/google-chrome.gpg] http://dl.google.com/linux/chrome/deb/ stable main | tee /etc/apt/sources.list.d/google-chrome.list
apt update
apt install google-chrome-stable -y

# install flaresolverr
RELEASE=$(wget -q https://github.com/FlareSolverr/FlareSolverr/releases/latest -O - | grep "title>Release" | cut -d " " -f 4)
wget -q https://github.com/FlareSolverr/FlareSolverr/releases/download/$RELEASE/flaresolverr_linux_x64.tar.gz
tar -xzf flaresolverr_linux_x64.tar.gz -C /opt
rm flaresolverr_linux_x64.tar.gz

# run as a service
cat <<EOF >/etc/systemd/system/flaresolverr.service
[Unit]
Description=FlareSolverr
After=network.target
[Service]
SyslogIdentifier=flaresolverr
Restart=always
RestartSec=5
Type=simple
Environment="LOG_LEVEL=info"
Environment="CAPTCHA_SOLVER=none"
WorkingDirectory=/opt/flaresolverr
ExecStart=/opt/flaresolverr/flaresolverr
TimeoutStopSec=30
[Install]
WantedBy=multi-user.target
EOF

systemctl enable -q --now flaresolverr.service
```

**Reply by namadori** [reply]
Thank you, I followed your instruction and it worked. I had to update the Jackett container before, as it was still on Debian 11 and there were some unmet dependencies.

It would be great to have this as an option on the Jackett container, just as the GUI is on the Wireguard container.

**Reply by remz1337** [reply]
Awesome. I have opened a PR based on that script (to setup as a separate LXC)
https://github.com/tteck/Proxmox/pull/3124

**Comment by Simpuhl** [comment]
I'd also like this, I get that you can add it to your prowlarr or jacket containers but sometimes its nice to keep things separated
